engine/class_modules/warlock/sc_warlock.hpp

```diff
@@ -508,6 +510,9 @@ struct warlock_t : public player_t
     gain_t* baleful_invocation;  // BFA - Azerite
 
     gain_t* memory_of_lucid_dreams;  // BFA - Essence
+
+    // SL
+    gain_t* scouring_tithe;
```

Comment:
As far as I can tell, the gain variable is not initialized anywhere.

Reply:
Nice catch, the init was deleted during a rebase!